package.json

```diff
@@ -13,7 +13,7 @@
     "fix:es": "eslint --color --fix --cache --cache-location node_modules/.cache/eslint/ src",
     "fix:format": "prettier --log-level warn -w .",
     "start-dev": "http-server ./dist --port 3500 --cors --proxy http://localhost:3500?",
-    "start-dev-https": "http-server ./dist --ssl --cert private.pem --key private.key --port 3500 --cors --proxy https://localhost:3500?",
+    "start-dev-https": "http-server ./dist --ssl --cert /Users/gagnp/keys/private.pem --key /Users/gagnp/keys/private.key --port 3500 --cors --proxy https://localhost:3500?",
```

Comment:
Shouldn't we get rid of `Users/gagnp/` here?

Reply:
sorry, I will get rid of that